src/components/DropZone/tests/DropZone.test.tsx

```diff
@@ -264,7 +264,7 @@ describe('<DropZone />', () => {
       expect(captionText.contains(overlayText)).toBe(true);
     });
 
-    it('renders a DisplayText containing the overlayText on extra-large screens', () => {
+    it('renders a DisplayText containing the overlayText when the bounding container is an extra large size', () => {
       setBoundingClientRect('extraLarge');
```

Comment:
Should these all be using the enum here?

Reply:
+1

Reply:
```
Element.prototype.getBoundingClientRect;
const widths = {	const widths = {
  small: 99,	  small: 99,
  medium: 159,	  medium: 159,
  large: 299,	  large: 299,
  extraLarge: 1024,	  extraLarge: 1024,
};	};
```

These don't quite correspond.

Reply:
I'm not sure an enum would work here unless I created a new one and not import the `Size` from types. These sizes (99, 159, 299) are all less than the bounding size which renders which overlay gets shown however the Size enum used in DropZone is to determine the breaking point when they change (100, 160, 300) by comparing values less then them. (The extralarge doesn't matter really what number it is because its the default). I might be misinterpreting the goal here though. Can you clarify if I should be creating a new enum or using the existing one?   For example replace `small: 99` with `small: 100` and you see the tests fail.

Reply:
I would create a new enum.